generators/entity-client/templates/angular/src/main/webapp/app/entities/entity-management.component.html.ejs

```diff
@@ -45,10 +45,10 @@
     </div>
     <%_ } _%>
     <br/>
-    <div class="alert alert-warning" *ngIf="<%=entityInstancePlural %> && <%=entityInstancePlural %>.length === 0">
+    <div class="alert alert-warning" *ngIf="<%=entityInstancePlural %>.length === 0">
```

Comment:
you forgot the `?` before the `.length` 😄 
See : https://blog.fullstacktraining.com/question-mark-in-angular-expression/

Reply:
Oh, I didn't known this syntax ! I was wondering why you made me changing the condition, but it explains why :smile: